assets/quick-order-list.js

```diff
@@ -150,19 +151,33 @@ class QuickOrderList extends HTMLElement {
         id: 'quick-order-list-total',
         section: document.getElementById(this.quickOrderListId).dataset.id,
         selector: '.quick-order-list__total'
+      },
+      {
+        id: 'CartDrawer',
+        selector: '#CartDrawer',
+        section: 'cart-drawer'
       }
     ];
   }
 
   renderSections(parsedState) {
     this.getSectionsToRender().forEach((section => {
       const sectionElement = document.getElementById(section.id);
+      if (sectionElement.parentElement.classList.contains('drawer')) {
+        parsedState.items.length > 0 ? sectionElement.parentElement.classList.remove('is-empty') : sectionElement.parentElement.classList.add('is-empty');
+
+        setTimeout(() => {
+          document.querySelector('#CartDrawer-Overlay').addEventListener('click', this.cart.close.bind(this.cart));
+          this.cart.open();
```

Comment:
I approve, but let's account for what Ed and Nick say about this 👆 looks a bit annoying to see a cart drawer every time.